app/application/controller.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    title: null,
```

Comment:
This property is missing documentation/comment. Lets do a pass and add to all properties.